app/home/templates/includes/property_listing_daily.html

```diff
@@ -9,37 +9,58 @@
+                                {% endfor %}
                                 <div class="card-body">
                                     <nav aria-label="breadcrumb">
                                         <ol class="breadcrumb">
-                                            <li class="breadcrumb-item active" aria-current="page"><i class="feather icon-home"></i> Location {{ property.location }}</li>
+                                            <li class="breadcrumb-item active" aria-current="page"><i class="feather icon-home"></i> Location: {{ property.location }}</li>
                                         </ol>
                                     </nav>
 
                                     <nav aria-label="breadcrumb">
                                         <ol class="breadcrumb">
-                                            <li class="breadcrumb-item active" aria-current="page"><i class="feather icon-home"></i> Price {{  property.price }}</li>
+                                            <li class="breadcrumb-item active" aria-current="page"><i class="feather icon-home"></i> Price: {{  property.price }}</li>
                                         </ol>
                                     </nav>
 
                                     <nav aria-label="breadcrumb">
                                         <ol class="breadcrumb">
-                                            <li class="breadcrumb-item"><a href="#!"><i class="feather icon-home"></i></a>Posted {{ property.date.strftime("%m/%d/%Y") }} at {{ property.date.strftime("%H:%M:%S")  }}</li>
+                                            <li class="breadcrumb-item"><a href="#!"><i class="feather icon-home"></i></a>Posted: {{ property.date.strftime("%m/%d/%Y") }} at {{ property.date.strftime("%H:%M")  }}</li>
                                         </ol>
                                     </nav>
                                 </div>
                             </div>
                         </div>
+                        {% endif %}
                     {% endfor %}
                     
                         </div>
                     </div>
+                    <div class="container m-4">
+                      {% for page_num in properties.iter_pages(left_edge=1, right_edge=1, left_current=1, right_current=5) %}
+                        {% if page_num %}
+                            {% if properties.page == page_num %}
+                              <a class="btn btn-outline-info mb-4 disabled" href="{{ url_for('home_blueprint.index', page=page_num) }}">{{ page_num }}</a>
+                            {% else %}
+                              <a class="btn btn-info mb-4 " href="{{ url_for('home_blueprint.index', page=page_num) }}">{{ page_num }}</a>
+                            {% endif %}
+                        {% else %}
+                        {% endif %}
+                      {% endfor %}
+                    </div>
```

Comment:
lets take this code into a new templates called `includes/pagination.html` and add  in this section an
`{% include "includes/pagination.html" %}` anywhere you need to use pagination